cmd/mtc/main.go

```diff
@@ -601,38 +615,30 @@ func handleInspectCert(cc *cli.Context) error {
 	switch proof := c.Proof.(type) {
 	case *mtc.MerkleTreeProof:
 		path := proof.Path()
+		batch := &mtc.Batch{
+			CA:     params,
+			Number: tai.BatchNumber,
+		}
 
-		params, err := inspectGetCAParams(cc)
-		if err == nil {
-			anch := proof.TrustAnchor().(*mtc.MerkleTreeTrustAnchor)
-
-			batch := &mtc.Batch{
-				CA:     params,
-				Number: anch.BatchNumber(),
-			}
-
-			if anch.IssuerId() != params.IssuerId {
-				return fmt.Errorf(
-					"IssuerId doesn't match: %s ≠ %s",
-					params.IssuerId,
-					anch.IssuerId(),
-				)
-			}
-			aa := c.Assertion.Abridge()
-			root, err := batch.ComputeRootFromAuthenticationPath(
-				proof.Index(),
-				path,
-				&aa,
+		if !reflect.DeepEqual(tai.Issuer, params.Issuer) {
```

Comment:
Manual Equal function would be nice.

Reply:
See [e5c15c7](https://github.com/bwesterb/mtc/pull/5/commits/e5c15c702786b1c74a86b82dff44ef4f6464beb8)